File: daily_coding_workout/15-nov-2018/reverseNumber/reverNum.c
#include <stdio.h>
//#include <conio.h>
int main()
{
    int n, x;
    int rev = 0;
    printf("Enter a number ! \n");
    scanf("%d", &n);
    while (n != 0)
    {
        x = n % 10;
        rev = rev * 10 + x;
        n = n / 10;
    }
    printf("The reversed number is %d ", rev);
    return 0;
}

File: DailyCodingPuzzles/15-nov-2018/reverseNumber/reverNum.java
import java.util.Scanner;

public class reverNum {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Input number  !");
        int n = sc.nextInt();
        int rev = 0;
        while (n > 0) {
            int y = n % 10;
            rev = rev * 10 + y;
            n /= 10;
        }
        System.out.println("Reversed number is :" + rev);
    }
}
